### PR TITLE
Introduces the UploadButton

### PR DESCRIPTION
**Pull request by KaiVolland**
### Description:

This introduces the `UploadButton`. In advantage to the ant design Upload you can pass in any child.

This also adds `webpack-cli` as it is required to build the examples.

### CODE REVIEW

File: src/Button/UploadButton/UploadButton.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import PropTypes from 'prop-types';
+
+import  {
+  SimpleButton
+} from '../../index';
+
+import './UploadButton.less';
+
+/**
+ * Class representating an upload Button. Can be used as Wrapper if children
```
Comment (marcjansen) [inline src/Button/UploadButton/UploadButton.jsx]: lowercase `Button` and `Wrapper`, please

File: src/Button/UploadButton/UploadButton.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+ * are given. Otherwise a Simplebutton will be rendered (be aware that passing).
```
Comment (marcjansen) [inline src/Button/UploadButton/UploadButton.jsx]: … is this a complete sentence?

File: src/Button/UploadButton/UploadButton.example.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+layout: basic.hbs
+title: UploadButton example
+description: This is an example using `UploadButton`
+collection: Examples
+---
+
+This exmaple demonstrates the use of the `UploadButton`.
```
Comment (marcjansen) [inline src/Button/UploadButton/UploadButton.example.md]: ~exmaple~ => example

File: src/Button/UploadButton/UploadButton.less [hunk 4]
```diff
@@ -0,0 +1,21 @@
+.react-geo-uploadbutton{
```
Comment (marcjansen) [inline src/Button/UploadButton/UploadButton.less]: Missing space before `{`

File: src/Button/UploadButton/UploadButton.less [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+  input[type="file"] {
+    cursor: pointer;
+    width: 100%;
+    position: absolute;
+    top: 0;
+    right: 0;
+    margin: 0;
+    padding: 0;
+    opacity: 0;
```
Comment (marcjansen) [inline src/Button/UploadButton/UploadButton.less]: This renders fine in all relevant browsers?

File: src/Button/UploadButton/UploadButton.jsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  static propTypes = {
+    /**
+     * The className which should be added.
+     * @type {String}
+     */
+    className: PropTypes.string,
+
+    /**
+     * The className which should be added.
+     * @type {String}
```
Comment (annarieger) [inline src/Button/UploadButton/UploadButton.jsx]: Wrong docs

File: src/Button/UploadButton/UploadButton.less [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  }
+
+  ::-webkit-file-upload-button {
+    cursor:pointer;
```
Comment (annarieger) [inline src/Button/UploadButton/UploadButton.less]: Missing space after `:`

File: src/Button/UploadButton/UploadButton.jsx [hunk 8]
```diff
@@ -34,17 +34,19 @@ class UploadButton extends React.Component {
 
     /**
      * The className which should be added.
```
Comment (annarieger) [inline src/Button/UploadButton/UploadButton.jsx]: Docs still wrong :sunglasses:

File: src/Button/UploadButton/UploadButton.less (hunk 5 above)
Comment (KaiVolland) [reply src/Button/UploadButton/UploadButton.less]: Checkt with Chrome and IE.

**Pull request merged**